source/css/scss/app/pages/monthly-good.scss

```diff
@@ -14,47 +14,32 @@
 				margin: 0;
 				font-size: rem-calc(38);
 				line-height: rem-calc(38);
-
-				@media #{$medium-up} {
-					font-size: $section-head-font-size;
-					line-height: rem-calc(28);
-				}
-
-				@media #{$large-up} {
-					font-size: rem-calc(46);
-					line-height: rem-calc(46);
-				}
 			}
 
 			.mg-subhead {
 				text-align: left;
 				margin: 0;
+				padding-top: rem-calc(5);
 				font-size: rem-calc(20);
 				line-height: $small-line-height;
-				padding-top: rem-calc(10);
 
 				@media (min-width: $x-small-breakpoint +1) and (max-width: $small-breakpoint) {
 					font-size: $headline-font-size-small-reduction;
 					line-height: rem-calc(26);
 					padding-top: rem-calc(15);
 				}
 
-				@media #{$medium-up} {
-					font-size: $slide-overlay-attribution-small-size;
-					line-height: rem-calc(16);
-					max-width: rem-calc(350);
-					padding-top: rem-calc(10);
+				@media (min-width: 580px) and (max-width: $large-breakpoint) {
+					font-size: $headline-font-size-small-reduction;
```

Comment:
if this works, great, but generally the "reduction" variables are used to subtract from other variables to calculate a reduced size (no need to change, just FYI)